### DISCUSSION TITLE
Jan: Simple Jan-based roleplay App without backend setup

### DISCUSSION THREAD

**Discussion by eckartal** [janhq/jan#3696] [Feature Ideas]
## Overview
Setting up roleplay apps like SillyTavern with separate backends can be tricky, and some users are unhappy with Backyard AI's recent updates. **There's a need for a simpler solution.**

## Goal
Develop a feature based on Jan that works out-of-the-box, without requiring a complex backend setup.

### Appendix
"Now I wish someone had implemented a roleplay app based on Jan to make it easier for those who don't want to run SillyTavern + separate backend (which can get finicky) or who don't agree with Backyard AI latest changes."

https://www.reddit.com/r/LocalLLaMA/comments/1fjpm7j/comment/lnprru6/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by progmars** [comment]
Totally support this. We need an open-source, community-driven alternative to commercial apps. Ok, we have SillyTavern and it is powerful, but the learning curve is quite steep and the UI feels a bit weird sometimes (the input field height jumping around etc.).
Maybe we wouldn't even need a separate app but just an extension, to add a UI area for managing character cards, if that's possible to achieve with extensions alone.
